tests/apichecks/testdata/exceptions/alpha-missingfields.txt

```diff
@@ -274,15 +274,8 @@
 [missing_field] crd=apphubdiscoveredservices.apphub.cnrm.cloud.google.com version=v1alpha1: field ".spec.projectRef" is not set; neither 'external' nor 'name' are set
 [missing_field] crd=apphubdiscoveredworkloads.apphub.cnrm.cloud.google.com version=v1alpha1: field ".spec.location" is not set in unstructured objects
 [missing_field] crd=apphubdiscoveredworkloads.apphub.cnrm.cloud.google.com version=v1alpha1: field ".spec.projectRef" is not set; neither 'external' nor 'name' are set
-[missing_field] crd=assetfeeds.asset.cnrm.cloud.google.com version=v1alpha1: field ".spec.assetNames[]" is not set in unstructured objects
-[missing_field] crd=assetfeeds.asset.cnrm.cloud.google.com version=v1alpha1: field ".spec.condition.description" is not set in unstructured objects
-[missing_field] crd=assetfeeds.asset.cnrm.cloud.google.com version=v1alpha1: field ".spec.condition.expression" is not set in unstructured objects
-[missing_field] crd=assetfeeds.asset.cnrm.cloud.google.com version=v1alpha1: field ".spec.condition.location" is not set in unstructured objects
-[missing_field] crd=assetfeeds.asset.cnrm.cloud.google.com version=v1alpha1: field ".spec.condition.title" is not set in unstructured objects
-[missing_field] crd=assetfeeds.asset.cnrm.cloud.google.com version=v1alpha1: field ".spec.contentType" is not set in unstructured objects
 [missing_field] crd=assetfeeds.asset.cnrm.cloud.google.com version=v1alpha1: field ".spec.folderRef" is not set; neither 'external' nor 'name' are set
```

Comment:
Looks like _more_ complete coverage :-) I don't know if we want to require folderRef / organizationRef for beta promotion - on the one hand it's a completely different code path that must be implemented manually (I believe - though we should definitely have a helper function), on the other hand it's a lot of work

Reply:
Yes, and labels. I want to tackle that separately